packages/gatsby/src/bootstrap/load-plugins/__tests__/load-plugins.ts

```diff
@@ -14,7 +14,7 @@ jest.mock(`gatsby-cli/lib/reporter`, () => {
     log: jest.fn(),
     warn: jest.fn((...args) => {
       // filter out compatible warnings as we get a lot of
-      // Plugin X is not compatible with your gatsby version 4.X.Y-next.Z - It requires gatsby@^5.0.0-alpha-v5
+      // Plugin X is not compatible with your gatsby version X - It requires X
```

Comment:
At this point this is probably not needed, but probably good to keep it for when we start working on next major. Thanks for comment update